### PR TITLE
Added the Footer.

### PR DESCRIPTION
**Pull request by Abhishekchauhan089**
@Logan1x ,  i have made the changes. Footer is added to your project

### CODE REVIEW

**Comment by Logan1x** [comment]
Can you show me how is it looking after the update?  @Abhishekchauhan089

**Comment by Logan1x** [comment]
I checked it by running locally on my machine and I want you to make some changes.
1. color scheme : Black(`#000000`) with yellow font(`#ffba08`).
2. We don't need that much content, we just need one line, `This project is an open source project, checkout our code here`(this repo hyperlink here)
3. Don't change the style.css reference in HTML code, it needed by flask to render properly.

Sorry I should've mentioned point number 1 & 2 before.

File: templates/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
           integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
-    <link rel='stylesheet' href="{{ url_for('static', filename='css/style.css') }}">
+    <link rel="stylesheet" type="text/css" href="../static/css/style.css"
```
Comment (Logan1x) [inline templates/index.html:11]: need to replace it back

**Review by Logan1x — CHANGES_REQUESTED** [review]

**Comment by Abhishekchauhan089** [comment]
@Logan1x , wait for a while.  after changing colors I will give you screenshot of the updates

**Comment by Abhishekchauhan089** [comment]
Screenshot (3)

**Comment by Abhishekchauhan089** [comment]
@Logan1x , Is it okay..

**Comment by Logan1x** [comment]
This is what it is looking after git pull on your repo

main page


image

after inputting a value and submitting


image


When I changed color from yellow to `#ffba08` i saw the color in font and the panel for footer is taking almost half of the page. Can we minimize it to as little as we can. And remove the `nucleotide` form the footer too.

**Comment by Logan1x** [comment]
Can you try running your repo with flask and then try looking at it? If you are not aware of how to do that please ask.

**Comment by Abhishekchauhan089** [comment]
@Logan1x yaa, i don't know how to run repo with flask

**Comment by Abhishekchauhan089** [comment]
@Logan1x , check this out again. i have made some changes

**Comment by Abhishekchauhan089** [comment]
Screenshot (4)

**Comment by Abhishekchauhan089** [comment]
Screenshot (5)

**Comment by Abhishekchauhan089** [comment]
@Logan1x , I think you might like this time, i have run this repo with flask

**Comment by Logan1x** [comment]
Thanks for doing so much work, I want to request one more change, Can we make the margin a bit less around 1? It feels like we are wasting so much space.

**Comment by Logan1x** [comment]
And you did great job by running the repo with flask :)

**Comment by Abhishekchauhan089** [comment]
@Logan1x I have tried for the margin, little bit of changes are there, but if we force to cut the margin more. then it will we no more responsive

**Comment by Logan1x** [comment]
@mrinal-27 Can you help here?

**Comment by mrinal-27** [comment]
@Logan1x  do we wanna change the margin or padding?

**Comment by mrinal-27** [comment]
> @Logan1x do we wanna change the margin or padding?

I mean, is the margin to be reduced between different elements or the padding for the footer has to be changed?

**Comment by Logan1x** [comment]
I want the footer to be a bit small in height. So whatever suits the purpose.

**Comment by mrinal-27** [comment]
@Logan1x  okay I'm on it. I have never run flask before so figuring that out now.

**Comment by Abhishekchauhan089** [comment]
Screenshot (6)

**Comment by Abhishekchauhan089** [comment]
Screenshot (7)

**Comment by Abhishekchauhan089** [comment]
@Logan1x is it okay..?

**Comment by Logan1x** [comment]
Much better than before, but if you can see there is space below the font.

**Comment by Abhishekchauhan089** [comment]
space below which font. please explain in brief

**Comment by Abhishekchauhan089** [comment]
that space below copyright?

**Comment by Logan1x** [comment]
I forgot about copyright, just saw you haven't scroll down to the bottom, So I think we can remove copyright thing and just this line in center. How about that? 
This will make your work much easier.

**Comment by Abhishekchauhan089** [comment]
Screenshot (8)

**Comment by Abhishekchauhan089** [comment]
Screenshot (9)

**Comment by Abhishekchauhan089** [comment]
Screenshot (10)

**Comment by Abhishekchauhan089** [comment]
were you saying something like this

**Comment by Logan1x** [comment]
Yes, yes that's much better. Please push this to your GitHub and I'll review it.

**Comment by Abhishekchauhan089** [comment]
@Logan1x hey, i have made the PR. check it out

**Review by Logan1x — APPROVED** [review]

**Comment by Logan1x** [comment]
@Abhishekchauhan089 Thank you for contributions :)

**Comment by Logan1x** [comment]
@Abhishekchauhan089 The Pr is merged but the changes I see on website https://nucleotide-generator.herokuapp.com/ is bit different. 

image

**Comment by Logan1x** [comment]
And the link hyperlink doesn't contain repo link.

**Comment by Abhishekchauhan089** [comment]
actually i haven't put the repo link...you just have to put your repository link..

**Comment by Abhishekchauhan089** [comment]
you have to replace #link with your repository link

**Pull request merged**